src/containers/navigation-icons/AccountIcon.jsx

```diff
@@ -0,0 +1,46 @@
+import Avatar from '@mui/material/Avatar'
+import Tooltip from '@mui/material/Tooltip'
+import { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { userService } from '~/services/user-service'
+import { styles } from '~/containers/navigation-icons/NavigationIcons.styles'
+import { useTranslation } from 'react-i18next'
+
+const AccountIcon = ({ openMenu }) => {
+  const { t } = useTranslation()
+  const store = useSelector((state) => state.appMain)
+  const [firstName, setFirstName] = useState('')
+  const [lastName, setLastName] = useState('')
+  const [photo, setPhoto] = useState('')
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      const userData = await userService.getUserById(
+        store.userId,
+        store.userRole
+      )
+      setPhoto(userData.data.photo)
+      setFirstName(userData.data.firstName)
+      setLastName(userData.data.lastName)
+    }
+
+    fetchUserData()
+  }, [store.userId, store.userRole])
+
+  const userNameForAvatar = firstName.charAt(0) + lastName.charAt(0)
+
+  return (
+    <Tooltip title={t('iconsTooltip.account')}>
+      <Avatar
+        alt='User'
+        onClick={openMenu}
+        src={photo}
+        sx={styles.studentIcons}
+      >
+        {userNameForAvatar}
+      </Avatar>
+    </Tooltip>
```

Comment:
Did you check what we will see if BE not available or we have a slow internet connection? userNameForAvatar  will empty string and photo also is empty string.

Reply:
I've chacked, if slow internet connection, we can see the same result, but not so fast
If remowe userNameForAvatar and photo, user can see this picture

![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/127191734/def9bedc-7e82-45ec-8496-a9f4df1e0cb9)